lib/chartmogul/concerns/pageable_with_anchor.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module Concerns
+    module PageableWithAnchor
```

Comment:
[improvement] You're introducing this new concern. Something like `.next_page` support method in the SDK could make the life of users easier, otherwise the user has to manually take the last ID, put it into filter and make another request etc. WDYT?

Reply:
Yes, we were discussing a auto pagination where `.find_all` actually fetches all records using appropriate pagination strategy. But we were not able to make that happen for this quarter. I will put that in the backlog.